dotcom-rendering/src/lib/image.ts

```diff
@@ -53,7 +55,10 @@ export const generateImageURL = ({
 		s: 'none',
 	});
 
+	// Only append the crop parameter if it's provided
+	const cropParam = aspectRatio ? `&crop=${aspectRatio}` : '';
+
 	return `https://i.guim.co.uk/img/${getServiceFromUrl(url)}${
 		url.pathname
-	}?${params.toString()}`;
+	}?${params.toString()}${cropParam}`;
```

Comment:
Ideally, this would be part of the urlsearchparams above but when doing that, Fastly-image-service seems unable to decode the ":". If anyone has any thoughts on how to solve this I'd be very keen to chat.